com.unity.cinemachine/Runtime/Components/CinemachineOrbitalFollow.cs

```diff
@@ -440,9 +442,9 @@ public override void MutateCameraState(ref CameraState curState, float deltaTime
             if (HorizontalAxis.Recentering.Enabled)
                 UpdateHorizontalCenter(orient);
 
-            HorizontalAxis.UpdateRecentering(deltaTime, gotInput);
-            VerticalAxis.UpdateRecentering(deltaTime, gotInput);
-            RadialAxis.UpdateRecentering(deltaTime, gotInput);
+            HorizontalAxis.UpdateRecentering(deltaTime, gotInputX);
+            VerticalAxis.UpdateRecentering(deltaTime, gotInputY);
+            RadialAxis.UpdateRecentering(deltaTime, gotInputZ);
```

Comment:
I get the idea I think it's a great one but in some cases you want all the axis to reset. Is it possible to give a choice (Maybe an inspector property)?

Reply:
Can you give an example?